pdp10-kl build sometimes has issues booting ITS from DDT

I've run into this issue a couple times now. When booting ITS the first time after getting ITS BIN, SALV BIN, and @ DDT copied to the root volume, expect times out with output like this:
```
spawn ./tools/sims/BIN/pdp10-ka build/pdp10-kl/boot                                                                                                                                                                                                                                                                                                                                     

KA-10 simulator V4.0-0 Current        git commit id: $Format:                                                                                                                               
/media/eswenson/ITS Volume/ITS/ws/its/build/pdp10-kl/boot-12> set dc dis                                                                                                                    
Missing value                                                                                                                                                                                                                                                                                                                                                                           
$L ITS BIN                                                                                                                                                                                                                                                                                                                                                                              
LSALV U   BINU                                                                                                                                                                              
The last command timed out.                                                                                                                                                                 
Makefile:100: recipe for target 'out/pdp10-kl/rp04.1' failed                                                                                                                                
make: *** [out/pdp10-kl/rp04.1] Error 1                                                                                                                                                     
```
If you invoke the emulator with the same boot script manually, and enter the two DDT commands with a pause before the second $$L SALV BIN command, things seem to work fine.  I think we need a delay between the invocation of those two commands.

--- a/build/kl10/include.tcl
+++ b/build/kl10/include.tcl
@@ -4,6 +4,7 @@ proc start_dskdmp_its {} {
     sleep 3
     respond "\n" "\033l"
     respond " " "its bin\r"
+    sleep 2
     respond "\n" "\033\033l"
     respond " " "salv bin\r"
     respond "\n" "\033y"

